src/sa_web/views.py

```diff
@@ -356,24 +356,44 @@ def readonly_file_api(request, path, datafilename='data.json'):
             raise Http404
 
 
-def api(request, path):
+def api(request, path, **kwargs):
     """
     A small proxy for a Shareabouts API server, exposing only
     one configured dataset.
     """
-    root = settings.SHAREABOUTS.get('DATASET_ROOT')
+
+    dataset_id = ""
+
+    #root = settings.SHAREABOUTS.get('DATASET_ROOT')
```

Comment:
We should remove  the commented-out code here, since it is no longer needed and may confuse a future programmer. Any commented-out code related to our old query params should definitely be removed too.